plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/MigrateProcedure.java

```diff
@@ -307,12 +309,21 @@ private Schema toIcebergSchema(List<Column> columns)
         return new Schema(icebergSchema.asStructType().fields());
     }
 
-    private static org.apache.iceberg.types.Type toIcebergType(Type type, AtomicInteger nextFieldId)
+    private static org.apache.iceberg.types.Type toIcebergType(Type type, AtomicInteger nextFieldId, HiveStorageFormat storageFormat)
     {
         if (type instanceof ArrayType || type instanceof MapType || type instanceof RowType) {
             // TODO https://github.com/trinodb/trino/issues/17583 Add support for these complex types
             throw new TrinoException(NOT_SUPPORTED, "Migrating %s type is not supported".formatted(type));
         }
+
+        // TODO: Add support for structural types with timestamp(3)
+        if (type.equals(TimestampType.TIMESTAMP_MILLIS)) {
+            // TODO Adding support for Avro is currently suspended as there are parallel works on switching to Avro native reader. https://github.com/trinodb/trino/pull/17221
```

Comment:
```
// TODO (some issue link): support Avro
```

where "some issue link" is a link to Trino issue which in turns links to https://github.com/trinodb/trino/pull/17221 as dependency